playback/service/src/main/java/de/danoeh/antennapod/playback/service/internal/ExoPlayerWrapper.java

```diff
@@ -101,7 +110,50 @@ private void createPlayer() {
                 DefaultLoadControl.DEFAULT_BUFFER_FOR_PLAYBACK_AFTER_REBUFFER_MS);
         loadControl.setBackBuffer((int) TimeUnit.MINUTES.toMillis(5), true);
         trackSelector = new DefaultTrackSelector(context);
-        exoPlayer = new ExoPlayer.Builder(context, new DefaultRenderersFactory(context))
+        final int skipSilenceDurationUs;
+        switch (skipSilence) {
+            case MILD:
+                skipSilenceDurationUs = 300_000;
+                break;
+            case MEDIUM:
+                skipSilenceDurationUs = 250_000;
+                break;
```

Comment:
Medium and mild seem very close together. Does this actually make a difference? I cannot hear a difference even between aggressive and mild. Do you have an example podcast where this makes a difference?